pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -222242,19 +222261,18 @@ self: {
        version = "0.8.0.0";
        sha256 = "1ylsr092a42xsbx6aancbcnfp1m66azv3vn44hqar069c5y96ilc";
        libraryHaskellDepends = [
-         async base compact constraints containers contravariant exceptions
+         async base compact constraints_0_12 containers contravariant exceptions
          ghc-prim mtl polysemy random reflection streaming text transformers
        ];
        testHaskellDepends = [
-         async base compact constraints containers contravariant exceptions
+         async base compact constraints_0_12 containers contravariant exceptions
          ghc-prim hspec mtl polysemy polysemy-plugin random reflection
          streaming text transformers
        ];
        testToolDepends = [ hspec-discover ];
        description = "Experimental, user-contributed effects and interpreters for polysemy";
        license = lib.licenses.bsd3;
        hydraPlatforms = lib.platforms.none;
-       broken = true;
```

Comment:
In order to mark this as unbroken, you'll need to remove `polysemy-zoo` from the broken list in:

 https://github.com/NixOS/nixpkgs/blob/599582499c548c8ad7f95dc0b2cb4d276795abeb/pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml

Reply:
Thanks a lot for the instructions, they were very clear. Are the changes OK?

I don't see how to change the branch that the PR is coming from, though, so I pushed `kephas:haskell-updates` to `kephas:master`.